content/SCALE/SCALEUIReference/Storage/_index.md

```diff
@@ -155,83 +155,83 @@ For example, a system with 18,446,744,073,709,551,615 bytes reports the number a
+ To schedule a single or a regular pool scrub operation, click **View All Scrub Tasks** to open the **Data Protections > Scrub Tasks** details screen and add or manage scrub tasks configured on your system.
 
-  {{< trueimage src="/images/SCALE/DataProtection/ScrubPoolDialog.png" alt="Scrub Pool Dialog" id="Scrub Pool Dialog" >}}
+ {{< trueimage src="/images/SCALE/DataProtection/ScrubPoolDialog.png" alt="Scrub Pool Dialog" id="Scrub Pool Dialog" >}}
 
-  **Confirm** activates the **Start Scrub** button.
-  {{< /expand >}}
+  **Confirm** activates the **Start Scrub** button.
+ {{< /expand >}}
 
 * {{< expand "Auto TRIM Dialog" "v" >}}
-  The **Edit Auto TRIM** option on the **ZFS Health** widget opens a dialog to set **Auto TRIM**.
+ The **Edit Auto TRIM** option on the **ZFS Health** widget opens a dialog to set **Auto TRIM**.
 
-  {{< trueimage src="/images/SCALE/Storage/PoolOptionsAuotTRIM.png" alt="Pool Option Auto TRIM" id="Pool Option Auto TRIM" >}}
+ {{< trueimage src="/images/SCALE/Storage/PoolOptionsAuotTRIM.png" alt="Pool Option Auto TRIM" id="Pool Option Auto TRIM" >}}
 
-  When enabled, **Auto TRIM** allows TrueNAS to periodically review data blocks and identify which empty blocks of obsolete blocks it can delete.
-  Leave unselected to incorporate day block overwrites when a device write is started (default).
-  Select **Confirm** to activate **Save**.
+ When enabled, **Auto TRIM** allows TrueNAS to periodically review data blocks and identify which empty blocks of obsolete blocks it can delete.
+ Leave unselected to incorporate day block overwrites when a device write is started (default).
+ Select **Confirm** to activate **Save**.
 
-  For more details about TRIM in ZFS, see the `autotrim` property description in [zpool.8](https://zfsonlinux.org/manpages/0.8.1/man8/zpool.8.html).
-  {{< /expand >}}
+ For more details about TRIM in ZFS, see the `autotrim` property description in [zpool.8](https://zfsonlinux.org/manpages/0.8.1/man8/zpool.8.html).
+ {{< /expand >}}
 
 * {{< expand "Prune Deduplication Table Dialog" "v" >}}
-  The **Prune Deduplication Table** dialog shows pruning measurement options the system should use when pruning the deduplication table (DDT).
-  Options are **Percentage** and **Age**.
-  
-  **Percentage** shows a slider to set the size (maximum amount of storage) percentage threshold the DDT is filled before it reaches the maximum size.
-  When reached the system prunes the table. 
+ The **Prune Deduplication Table** dialog shows pruning measurement options the system should use when pruning the deduplication table (DDT).
+ Options are **Percentage** and **Age**.
+  
+  **Percentage** shows a slider to set the size (maximum amount of storage) percentage threshold the DDT is filled before it reaches the maximum size.
+ When reached the system prunes the table. 
```

Comment:
This looks a little odd in my local build test, making a suggestion to see if that helps, but local build test and see what you think.
![image](https://github.com/user-attachments/assets/f95e2c8c-dc18-4671-b113-a43f7d665d59)

```suggestion
 The **Prune Deduplication Table** dialog shows pruning measurement options the system should use when pruning the deduplication table (DDT).
 Options are **Percentage** and **Age**.
  
 **Percentage** shows a slider to set the size (maximum amount of storage) percentage threshold the DDT is filled before it reaches the maximum size.
 When reached the system prunes the table. 
```